src/core/apiState.ts

```diff
@@ -85,13 +102,31 @@ export type MutationKeys<Definitions extends EndpointDefinitions> = {
 
 type BaseQuerySubState<D extends BaseEndpointDefinition<any, any, any>> = {
   originalArgs: QueryArgFrom<D>;
```

Comment:
@phryneas any recommendation for a description for `originalArgs` here?

Reply:
Hmm. Maybe something like
> argument originally passed into the hook or `initiate` action call

Reply:
Sounds good to me, thanks 👍